src/components/GlobalErrorBoundary/index.tsx

```diff
@@ -43,13 +43,29 @@ const LinkContent = styled.div`
   }
 `
 
-type Props = {
-  error: Error
-  componentStack: string | null
-  resetError: () => void
-}
+const GlobalErrorBoundaryFallback: FallbackRender = ({ error, componentStack }) => {
+  //  When loading app during release, chunk load failure may occur
+  const chunkFailedMessage = /Loading chunk [\d]+ failed/
+  const isChunkError = error?.message && chunkFailedMessage.test(error.message)
+
+  const time = new Date().getTime()
+
+  const lastReloadKey = 'lastReload'
+  const lastReloadString = localStorage.getItem(lastReloadKey)
+  const lastReload = lastReloadString && JSON.parse(lastReloadString)
+
+  // Expires 10 secs after last chunk failure reload
+  const isExpired = time > +lastReload + 10000
+
+  if (isChunkError && isExpired) {
+    localStorage.setItem(lastReloadKey, JSON.stringify(time))
+    window.location.reload()
+
+    return null
+  }
+
+  localStorage.removeItem(lastReloadKey)
```

Comment:
This logic will run each time the component is rendered. Is it intentional?

Reply:
The error boundary fallback component is only rendered when an error occurs, so I think it should be fine. I originally had this inside a side effect depending on the `error` object, but Sentry's fallback doesn't like the use of hooks. What are your thoughts?

Reply:
Makes sense, thanks